src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/PkgImportControlTest.java

```diff
@@ -69,6 +71,13 @@ public void setUp() {
         icRootRegexpParent.addChild(icBootRegexpParen);
     }
 
+    @Test
+    public void testRegexp() {
+        // . replaced with x to avoid matching with regexp meta-character .
```

Comment:
lets remove this comment and maybe use more descriptive test name

Reply:
done